argilla-sdk/src/argilla_sdk/_helpers/_log.py

```diff
@@ -23,6 +23,8 @@
     "critical": logging.CRITICAL,
 }
 
+logger = logging.getLogger(name=__name__)
```

Comment:
Does it make sense to tune the logger name to `"argilla"` or `"argilla.sdk"`? Logs can be confusing by providing this module name ("_log: ..."
```suggestion
logger = logging.getLogger(name="argilla")
```

Reply:
That makes sense. I'd say `argilla.sdk` then.